.github/workflows/beaker.yml

```
@@ -112,8 +115,9 @@ jobs:
         if: ${{ inputs.rubocop }}
       - name: Setup Test Matrix
         id: get-outputs
-        run: bundle exec metadata2gha --domain ${{ inputs.domain }} --pidfile-workaround ${{ inputs.pidfile_workaround }} --beaker-facter "${{ inputs.beaker_facter }}" --beaker-hosts "${{ inputs.beaker_hosts }}"
-
+        run: bundle exec metadata2gha --domain ${{ inputs.domain }} --pidfile-workaround
+          ${{ inputs.pidfile_workaround }} --beaker-facter "${{ inputs.beaker_facter }}"
+          --beaker-hosts "${{ inputs.beaker_hosts }}"
```

I don't think you can split the lines like that. Perhaps this is better:
```suggestion
        run: |
          bundle exec metadata2gha \
            --domain ${{ inputs.domain }}
            --pidfile-workaround ${{ inputs.pidfile_workaround }} \
            --beaker-facter "${{ inputs.beaker_facter }}" \
            --beaker-hosts "${{ inputs.beaker_hosts }}"
```

---

Oh yeah, I think I’d meant to put a `>` there. Should work with `>` I believe.

---

Above way would work too -- I tried putting it in with `>`; let me know if you have a preference. I ended up removing all the quoting to match the rest of this and fix the mixed quoting, but can also quote them all if you prefer.